### PR TITLE
mobile widgets docs

### PR DESCRIPTION
**Pull request by condrenjustin**
### What does this PR do?
adds documentation for monitor saved views widgets on mobile

### Motivation
GA of monitor saved views widgets

### Preview

https://docs-staging.datadoghq.com/justin/widgets/mobile/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/mobile/_index.md [hunk 1]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
 3. Give your incident a title, severity, and commander.
 
 
+## Widgets
+
+### Monitor Saved Views
+
+You can view your Monitor [Saved Views][18] from your home screen with Datadog widgets. Tap on any Saved View cell to open the “Monitor Search” screen in the app, with your Saved View already filled in.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:85]:
```suggestion
View your [monitor saved views][18] from your home screen with Datadog widgets. Tap on any saved view cell to open the "Monitor Search" screen in the app, with your saved view already filled in.
```

File: content/en/mobile/_index.md [hunk 2]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+**Note**: If you do not have any Monitor Saved Views, the widget will default to show you all Monitors.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:87]:
```suggestion
**Note**: If you do not have any monitor saved views, the widget shows you all monitors by default. 
```

File: content/en/mobile/_index.md [hunk 3]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+## Widgets
+
+### Monitor Saved Views
+
+You can view your Monitor [Saved Views][18] from your home screen with Datadog widgets. Tap on any Saved View cell to open the “Monitor Search” screen in the app, with your Saved View already filled in.
+
+**Note**: If you do not have any Monitor Saved Views, the widget will default to show you all Monitors. 
+
+#### Create a widget
+**iOS**:
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:90]: These would work better as tabs:

```
{{< tabs >}}
{{% tab "iOS" %}}

Content

{{% /tab %}}
{{% tab "Android" %}}

Content

{{% /tab %}}
{{< /tabs >}}
```

File: content/en/mobile/_index.md [hunk 4]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Long press on the home screen.
+Tap the “+” button on the top left corner of the screen.
+Search for “Datadog” widgets.
+Select your prefered size (medium allows up to 3 and large up to 6)
+Place it where you want on your screen.
+
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:96]: Add bullet points to all step lines

File: content/en/mobile/_index.md [hunk 5]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+You can view your Monitor [Saved Views][18] from your home screen with Datadog widgets. Tap on any Saved View cell to open the “Monitor Search” screen in the app, with your Saved View already filled in.
+
+**Note**: If you do not have any Monitor Saved Views, the widget will default to show you all Monitors. 
+
+#### Create a widget
+**iOS**:
+Long press on the home screen.
+Tap the “+” button on the top left corner of the screen.
+Search for “Datadog” widgets.
+Select your prefered size (medium allows up to 3 and large up to 6)
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:94]:
```suggestion
- Select your prefered size (medium allows up to 3 and large up to 6).
```

File: content/en/mobile/_index.md [hunk 6]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Place it where you want on your screen.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:95]:
```suggestion
- Place the widget in your desired, onscreen location.
```

File: content/en/mobile/_index.md [hunk 7]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+**Android**:
+Open the Android Widget Picker. Either long press your home screen, or long press the Datadog app icon.
+Tap on “Widgets”. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble. 
+Drag the Monitor Saved Views widget on your home screen.
+
+#### Edit a widget
+**iOS**:
+Long press on the widget to configure.
+Tap on “Edit Widget”.
+Tap on the individual Saved View cell to select and deselect.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:106]:
```suggestion
- Tap on the individual saved view cell to select and deselect.
```

File: content/en/mobile/_index.md [hunk 8]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+You can reorder them by drag and dropping each cell.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:107]:
```suggestion
- Reorder the views by dragging and dropping each cell.
```

File: content/en/mobile/_index.md [hunk 9]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Tap out of the widget to validate your selection and exit the configuration screen.
+
+**Android**:
+Tap on the widget title to configure.
+Tap on “Saved views”.
+Tap on the individual Saved View cell to select and deselect.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:113]:
```suggestion
- Tap on the individual saved view cell to select and deselect.
```

File: content/en/mobile/_index.md [hunk 10]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+You can reorder them by drag and dropping each cell.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:114]: - Reorder the views by dragging and dropping each cell.

File: content/en/mobile/_index.md [hunk 11]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Tap “Save” to validate your selection and exit the configuration screen.
+Also, you can scroll the widget to see more Saved Views. Long press and resize the widget to fit your preference.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:116]:
```suggestion
- Scroll the widget to see more saved views. Long press and resize the widget to fit your preference.
```

File: content/en/mobile/_index.md [hunk 12]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+#### Switch organizations
+**iOS**:
+All organizations you’ve logged-in are displayed in the configuration screen
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:120]:
```suggestion
All organizations you’ve logged-in are displayed in the configuration screen.
```

File: content/en/mobile/_index.md [hunk 13]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+**Android**:
+From the configuration screen, tap on “Organization”
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:123]:
```suggestion
- From the configuration screen, tap on "Organization".
```

File: content/en/mobile/_index.md [hunk 14]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Select a new organisation. You might need to sign in.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:124]:
```suggestion
- Select a new organization (you might need to sign in).
```

File: content/en/mobile/_index.md [hunk 15]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Tap “Saved Views”. Select Saved Views to display or leave it as is.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:125]:
```suggestion
- Tap "Saved Views". Select saved views to display or leave it as is.
```

File: content/en/mobile/_index.md [hunk 16]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Tap “Save”. The widget is now configured with Saved Views from a different organisation.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:126]:
```suggestion
- Tap "Save". The widget is now configured with saved views from a different organization.
```

File: content/en/mobile/_index.md [hunk 17]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+#### Delete a widget
+**iOS**:
+You can delete a widget pressing the “-” button at the top left of the widget when editing your home screen, or by long pressing on the widget and then selecting “Remove Widget”
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:130]:
```suggestion
Delete a widget pressing the "-" button at the top left of the widget when editing your home screen, or by long pressing on the widget and then selecting "Remove Widget".
```

File: content/en/mobile/_index.md [hunk 18]
```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+**Android**:
+You can delete a widget by long pressing, drag and dropping the Widget on the “Remove” button.
```
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:133]:
```suggestion
Delete a widget by long pressing, dragging, and dropping the widget on the "Remove" button.
```

File: content/en/mobile/_index.md (hunk 1 above)
Comment (ruthnaebeck) [inline content/en/mobile/_index.md:85]: Change all `”` to `"`

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: content/en/mobile/_index.md (hunk 3 above)
Comment (condrenjustin) [reply content/en/mobile/_index.md]: I think the commit I made fixes this, but I am not sure

File: content/en/mobile/_index.md [hunk 19]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Open the Android Widget Picker. 
+- Either long press your home screen, or long press the Datadog app icon.
+- Tap on "Widgets". If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble. 
+- Drag the Monitor Saved Views widget on your home screen.
+
```
Comment (apigirl) [inline content/en/mobile/_index.md:108]:
```suggestion

{{% /tab %}}
{{< /tabs >}}
```

File: content/en/mobile/_index.md [hunk 20]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Tap on the widget title to configure.
+- Tap on "Saved views".
+- Tap on the individual saved view cell to select and deselect.
+- Reorder the views by dragging and dropping each cell.
+- Tap "Save" to validate your selection and exit the configuration screen.
+- Scroll the widget to see more saved views. Long press and resize the widget to fit your preference.
+
```
Comment (apigirl) [inline content/en/mobile/_index.md:129]:
```suggestion

{{% /tab %}}
{{< /tabs >}}
```

File: content/en/mobile/_index.md [hunk 21]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+#### Switch organizations
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+All organizations you’ve logged-in are displayed in the configuration screen.
+
```
Comment (apigirl) [inline content/en/mobile/_index.md:136]:
```suggestion

{{% /tab %}}
```

File: content/en/mobile/_index.md [hunk 22]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+**Android**:
```
Comment (apigirl) [inline content/en/mobile/_index.md:137]:
```suggestion
{{% tab "Android" %}}

```

File: content/en/mobile/_index.md [hunk 23]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+- From the configuration screen, tap on "Organization".
+- Select a new organization (you might need to sign in).
+- Tap "Saved Views". Select saved views to display or leave it as is.
+- Tap "Save". The widget is now configured with saved views from a different organization.
+
```
Comment (apigirl) [inline content/en/mobile/_index.md:142]:
```suggestion
{{% /tab %}}
{{< /tabs >}}
```

File: content/en/mobile/_index.md [hunk 24]
```diff
@@ -80,6 +78,83 @@ On the Incidents page, you can view, search and filter all incidents that you ha
...
+Delete a widget pressing the "-" button at the top left of the widget when editing your home screen, or by long pressing on the widget and then selecting "Remove Widget".
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+Delete a widget by long pressing, dragging, and dropping the widget on the "Remove" button.
+
+
```
Comment (apigirl) [inline content/en/mobile/_index.md:156]:
```suggestion
{{% /tab %}}
{{< /tabs >}}
```

**Pull request merged**